### PR TITLE
fix: revise references to deprecated lambda node version update page

### LINKED ISSUE #4974: [missing-page] lambda. node version update seems to be missing
**Page**: [`https://docs.amplify.aws/cli/migration/lambda-node-version-update/`](https://docs.amplify.aws/cli/migration/lambda-node-version-update/)

**Feedback**:

### PR DESCRIPTION
**Pull request by josefaidt**
_Issue #, if available:_

closes #4974

_Description of changes:_

removes "read more" links pointing to a deprecated page

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/lib/auth/js/start.mdx [hunk 1]
```diff
@@ -22,7 +22,7 @@ amplify push
 
 A configuration file called `aws-exports.js` will be copied to your configured source directory, for example `./src`.
 
-> If your Authentication resources were created with Amplify CLI version 1.6.4 and below, you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. [Read more](/cli/migration/lambda-node-version-update)
+> If your Authentication resources were created with Amplify CLI version 1.6.4 and below, you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda.
```
Comment (ykethan) [inline src/fragments/lib/auth/js/start.mdx:25]: We can refer them to new page: https://docs.amplify.aws/cli/function/configure-options/

**Review by ykethan — APPROVED** [review]
LGTM

**Review by david-mcafee — APPROVED** [review]
LGTM! 🎉

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Review by katiegoines — APPROVED** [review]

**Pull request merged**